authentication/authentication-common/src/main/java/io/mosip/authentication/common/service/impl/hotlist/HotlistServiceImpl.java

```diff
@@ -53,4 +54,18 @@ public void unblock(String id, String idType) throws IdAuthenticationBusinessExc
 		}
 	}
 
+	@Override
+	public HotlistDTO getHotlistStatus(String id, String idType) throws IdAuthenticationBusinessException {
+		HotlistDTO dto = null;
+		Optional<HotlistCache> hotlistData = hotlistCacheRepo.findByIdHashAndIdType(id, idType);
```

Comment:
@mamta-344 If the entry not present in DB, please throw an IdAuthenticationBusinessException error saying "Invalid input parameter - ID / ID Type" You can add this in io.mosip.authentication.core.constant.IdAuthenticationErrorConstants.INVALID_INPUT_PARAMETER.

Reply:
sure i will change